plugins/extractors/maxcompute/maxcompute.go

```diff
@@ -240,7 +296,7 @@ func buildColumns(dataType datatype.DataType) []*v1beta2.Column {
 	return columns
 }
 
-func (e *Extractor) buildTableAttributesData(schemaName string, tableInfo *tableschema.TableSchema) map[string]interface{} {
+func (e *Extractor) buildTableAttributesData(schemaName string, _ *odps.Table, tableInfo *tableschema.TableSchema) map[string]interface{} {
```

Comment:
why not just omit the ` _ *odps.Table` as parameter since not used anyway?

Reply:
omitted